phase1/openstack/configure-vm.sh

```diff
@@ -23,3 +34,23 @@ apt-get update
 apt-get install -y docker-engine=1.12.0-0~xenial
 systemctl enable docker || true
 systemctl start docker || true
+
+ensure_gsutil_command(){
```

Comment:
why is this necessary for openstack?

Reply:
gsutil is used to download the packages from GCS. GCS is the current location where all the release packages are hosted.

Reply:
ah okay, makes sense